zanata-war/src/main/webapp-jboss/WEB-INF/classes/META-INF/persistence.xml

```diff
@@ -87,6 +87,8 @@
       <!-- Search engine -->
       <property name="hibernate.search.default.directory_provider"
         value="org.hibernate.search.store.impl.FSDirectoryProvider" />
+      <property name="hibernate.search.default.locking_strategy"
+        value="simple" />
```

Comment:
This is the default for FSDirectoryProvider correct? Are you just being explicit or is there something else?